core/common/src/main/scala/net/liftweb/common/Box.scala

```diff
@@ -793,6 +793,10 @@ sealed abstract class Box[+A] extends Product with Serializable{
     * Transforms this box using the `transformFn`. If `transformFn` is defined for this box,
     * returns the result of applying `transformFn` to it. Otherwise, returns this box unchanged.
     *
+    * If you want to change the content of a `Full` box, using [[map]] or [[collect]] might be better
+    * suited to that purpose. If you want to convert an `Empty`, `Failure` or a `ParamFailure` into a
+    * `Full` box, you should use [[flip]].
```

Comment:
I'd put backticks around `flip`, `map`, and `collect` (in addition to the `[[` and `]]`).

Reply:
Generally, as a part of the review, I would take the words `I'd put backticks` to mean that I (as the author) should make that change. But since you approved, I am guessing you are making that change? Let me know if I should do it, and I'll push the change.